http.go

```diff
@@ -4,16 +4,15 @@ import (
 	"fmt"
 	"io/ioutil"
 
-	em "github.com/dafiti/echo-middleware"
-	"github.com/labstack/echo"
-	"github.com/labstack/echo/middleware"
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/echo/v4/middleware"
 	"github.com/sirupsen/logrus"
 )
 
 func (om *OpenMock) startHTTP() {
 	e := echo.New()
 	e.HideBanner = true
-	e.Use(em.Logrus())
+	e.Use(middleware.Logger())
```

Comment:
changed to the official logger so that we can remove the dependency on `github.com/dafiti/echo-middleware`